src/app/container/info-tab/info-tab.component.ts

```diff
@@ -103,4 +109,15 @@ export class InfoTabComponent implements OnInit {
   cancelEditing(): void {
     this.infoTabService.cancelEditing();
   }
+
+
+  /**
+   * Returns a link to the primary descriptor for the given tool version
+   * @param path tool path
+   * @param versionName name of version
+   */
+  getTRSLink(path: string, versionName: string): string {
+    return `${Dockstore.API_URI}${ga4ghPath}/tools/${encodeURIComponent(path)}` +
+      `/versions/${encodeURIComponent(versionName)}/plain-CWL/descriptor`;
```

Comment:
looks like this only handles CWL right now. make PLAIN_CWL and PLAIN_WDL something that can easily be changed once swagger has it.

Reply:
Oops